docs/kqueen.rst

```diff
@@ -436,6 +444,13 @@ Provision a Kubernetes cluster using Amazon Elastic Kubernetes Service
 #. Click ``Submit``.
 #. To track the cluster status, navigate to the KQueen main dashboard.
 
+.. note::
+
+   Currently Kqueen does not support Kubernetes workers-management due Amazon EKS API restrictions,
```

Comment:
Currently, KQueen does not support Kubernetes workers-management because of the Amazon EKS API restrictions. To cover the Kubernetes workers use cases, see `Getting started with Amazon EKS <https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html>`_